packages/jazz-react/src/auth/MagicLinkAuth.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  MagicLinkAuthConsumerOptions,
+  MagicLinkAuthProviderOptions,
+  useCreateMagicLinkAuthAsConsumer as useCreateMagicLinkAuthAsConsumerCore,
+  useCreateMagicLinkAuthAsProvider as useCreateMagicLinkAuthAsProviderCore,
+  useHandleMagicLinkAuthAsConsumer as useHandleMagicLinkAuthAsConsumerCore,
+  useHandleMagicLinkAuthAsProvider as useHandleMagicLinkAuthAsProviderCore,
+} from "jazz-react-core";
+
+export function useCreateMagicLinkAuthAsProvider(
+  options?: Partial<MagicLinkAuthProviderOptions>,
+) {
+  return useCreateMagicLinkAuthAsProviderCore(window.location.origin, options);
+}
+
+export function useCreateMagicLinkAuthAsConsumer(
+  options?: Partial<MagicLinkAuthConsumerOptions>,
+) {
+  return useCreateMagicLinkAuthAsConsumerCore(window.location.origin, options);
+}
+
+export function useHandleMagicLinkAuthAsConsumer(
+  options?: Partial<MagicLinkAuthConsumerOptions>,
+) {
+  return useHandleMagicLinkAuthAsConsumerCore(
```

Comment:
On the web we should handle `hash` navigation.

When the user paste a link with a different hash but same path the page is not reloaded.
With the current implementation the auth would not start.